docs/getting-started/compute-distances.md

```diff
@@ -0,0 +1,114 @@
+This gives you great flexibility for cases with specific business logic, such as this one.
+
+We have one input and one output: therefore our custom class is a *CustomDfTransformer* (instead of CustomDf**s**Transformer).
+It takes the input dataframe *df* and calls a User Defined Function called *calculateDistanceInKilometerUdf*
+to do the computation.
+It expects the column names dep_latitude_deg, dep_longitude_deg, arr_latitude_deg and arr_longitude_deg in the input.
+This matches the column names we used in the SQL-Code in the last-step.
+
+We won't go into the details of the Udf. 
+You can follow this [stackoverflow link](https://stackoverflow.com/questions/27928/calculate-distance-between-two-latitude-longitude-points-haversine-formula)
+if you want to learn more.
+
+Finally, the transformation writes the result into a column called distance.
+It also adds a column *could_be_done_by_rail* to the output that simply checks if the distance is smaller than 500 km.
+
+In order to wire this CustomTransformation into our config, we add the following action:
+
+      compute_distances {
+        type = CopyAction
+        inputId = btl-departures-arrivals-airports
+        outputId = btl-distances
+        transformer.class-name = com.sample.ComputeDistanceTransformer
+        metadata {
+          feed = compute
+        }
+      }
+
+We used a CopyAction and told it to execute the code in the class *com.sample.ComputeDistanceTransformer* to transform the data.
+We could also have used a CustomSparkAction like in the previous step, 
+but this would have resulted in more complex code working with lists of inputs, outputs and transformers.
+
+
+## Try it out
+
+You can execute the usual *docker run* command :
+
+    docker run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/config:/mnt/config demo:latest -c /mnt/config --feed-sel .*
+
+Under *data/btl-distances* you can now see the final result. 
+
+### The Execution DAG
+
+In the console, you probably started noticing some pretty ASCII Art that looks like this:
+
+                         ┌─────┐
+                         │start│
+                         └┬──┬─┘
+                          │  │
+                          │  └───────────────────┐
+                          v                      │
+     ┌─────────────────────────────────────────┐ │
+     │select-airport-cols SUCCEEDED PT4.421793S│ │
+     └───────────────┬─────────────────────────┘ │
+                     │                           │
+                     v                           v
+     ┌──────────────────────────────────────────────┐
+     │join_departures_airports SUCCEEDED PT2.938995S│
+     └──────────────────────┬───────────────────────┘
+                            │
+                            v
+       ┌────────────────────────────────────────┐
+       │compute_distances SUCCEEDED PT1.160045S│
+       └────────────────────────────────────────┘
+
+This is the Execution *DAG* of our data pipeline. 
```

Comment:
Why are Actions download-airports and download-departures not executed?
It would be interesting to have them included here, as they can run in parallel.
We could then further explain that SDL runs Actions in parallel up to the limit set with parameter "--parallelism".

Reply:
I want to limit the number of times that the user downloads the files because I noticed that the API tends to block you if you use it too many times within a short period of time. I warn the reader about that in one of the first steps of the tutorial.
But I agree it's nicer to see the whole dag in the end...
Shall I replace it here?

Reply:
Yes, i would show the full DAG here.

Reply:
done. While doing it I noticed that the REST Service is just a bit slow to respond. I added the readTimeoutMs option to ext-departures, now it's more reliable. I also played around with ext-departures=0 in the tutorial to show a partially failed execution